recipes/mesmer/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "mesmer" %}
+{% set pypiname = "mesmer-emulator" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ pypiname[0] }}/{{ pypiname }}/{{ pypiname }}-{{ version }}.tar.gz
+  sha256: 529ec371032fa003b1af81a1dc608c7b86732610b84b51cdc475b902951c248d
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
```

Comment:
Upstream only lists >=3.7, not sure if that's important or not.

Reply:
oops good pick up 

```suggestion
    - python >=3.7
```